core/lib/tests/http_uri_serde.rs

```diff
@@ -44,15 +44,15 @@ fn uri_serde() {
 
 #[test]
 fn uri_serde_round_trip() {
-    let tmp = Figment::from(Serialized::default("default", UriContainer {
+    let tmp = Figment::from(Serialized::default("uris", UriContainer {
```

Comment:
Use [`Serialized::defaults()`](https://docs.rs/figment/0.10.5/figment/providers/struct.Serialized.html#method.defaults),

Reply:
Thanks, I have switched to that. I'm also fixing the CI failure (It looks like shield was added since I started, and I have updated it to reflect the slightly different Absolute and Reference Uris)